### ISSUE TITLE
Changing the origin of the skeleton component

### ISSUE THREAD

**Issue by mees-manusvr** [ValveSoftware/openvr#1369]
Hi!

I am currenlty working on a driver for a VR glove that measures finger curl. It uses a Vive Tracker for the position and orientation which is attached to the back of the hand, that means that the center of rotation is also on the back of the hand. However, the skeleton's origin defaults to a location somewhere around the thumb (I am assuming this represents the point where the vive controller and thumb would connect) According to [the documentation](https://github.com/ValveSoftware/openvr/wiki/Creating-a-Skeletal-Input-Driver#createskeletoncomponent) of OpenVR you can set the place of the origin to a place on the rendermodel by changing the pchBasePosePath parameter in the CreateSkeletonComponent function. 

`IVRDriverInput::CreateSkeletonComponent( PropertyContainerHandle_t ulContainer, const char *pchName, const char *pchSkeletonPath, const char *pchBasePosePath, EVRSkeletalTrackingLevel eSkeletalTrackingLevel, const VRBoneTransform_t *pGripLimitTransforms, uint32_t unGripLimitTransformCount, VRInputComponentHandle_t *pHandle )`

>pchBasePosePath - The path of the location on the controller model that the skeleton should use as its origin  

I assume this path points to a component in the render model's json file. Unfortunately, changing this path does not change anything for me. Taking the Vive Controller render model for example, this has a tip, base or handgrip component, all on different locations of the controller. Passing "/pose/base" for example does not change the origin. 

The result is that the orientation of the skeleton does not line up with the hand of the user and the center of rotation is different, meaning that if the user rotatest his wrist the skeleton rotation around his thumb instead of the wirst. In the pictures the difference in orientation between the two is demonstrated. I have a [small video](https://www.dropbox.com/s/neooxlxjicm4r54/20200430_111707.mp4?dl=0) showing the rotational problem that it creates.

To summarize, how can I change the origin of the skeleton? Am I misunderstanding the documentation or am I doing something else wrong? I hope someone has any tips!

All the best,
Mees

![RenderModel](https://user-images.githubusercontent.com/60879848/80979510-b765eb00-8e27-11ea-8c21-a9e0e1ff211a.JPG)

![Skeleton](https://user-images.githubusercontent.com/60879848/80979506-b6cd5480-8e27-11ea-87d3-303fac54a2c7.JPG)
